src/Controls/src/Core/Slider.cs

```diff
@@ -9,31 +8,23 @@ namespace Microsoft.Maui.Controls
 	public partial class Slider : View, ISliderController, IElementConfiguration<Slider>
 	{
 		/// <include file="../../docs/Microsoft.Maui.Controls/Slider.xml" path="//Member[@MemberName='MinimumProperty']/Docs" />
-		public static readonly BindableProperty MinimumProperty = BindableProperty.Create("Minimum", typeof(double), typeof(Slider), 0d, validateValue: (bindable, value) =>
-		{
-			var slider = (Slider)bindable;
-			return (double)value < slider.Maximum;
-		}, coerceValue: (bindable, value) =>
+		public static readonly BindableProperty MinimumProperty = BindableProperty.Create(nameof(Minimum), typeof(double), typeof(Slider), 0d, coerceValue: (bindable, value) =>
```

Comment:
If we remove the validations here, sould we include in the handlers a check for nonsensical states when any of the properties are updated? (avoid errors,  settings zeros etc)

Reply:
I think the only scenario that gets you in trouble right now is when you set the `Minimum` to more than the `Maximum`. From what I tested the thumb will just get stuck. And that is just the default behavior then for that platform. So if I was a iOS or Android developer and I were to do that, I would end up with the same thing. Should we then get in the middle and do something else?

The problem is then, again, the order of things. If we are going to make checks like: minimum is more than maximum and them we make minimum equal to maximum and your setting one before the other, you again end up with behavior you might not want and that is dependant on the order of declaration.

Reply:
Can we just have the mapping method set the platform min/max/value to zero if the values aren't valid? Then all the platforms do something consistent when the values don't make sense, and the order doesn't matter.

Reply:
I'm inclined not to do this, as the way the handlers are structured, if we want to update everything to 0 when one change is invalid, we may cause the UI to 'skip/jump' by changing the values temporarily until the next property change happens and brings in a set of 'valid' values again.

I think what this branch has now is probably reasonable enough.  It doesn't seem to crash any of the platforms so far (still have iOS to validate).